go.mod

```diff
@@ -6,8 +6,8 @@ require (
 	cuelang.org/go v0.3.2
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/bugsnag/bugsnag-go v1.5.0
-	github.com/envoyproxy/go-control-plane v0.10.2-0.20220325020618-49ff273808a1
-	github.com/envoyproxy/protoc-gen-validate v0.4.0
+	github.com/envoyproxy/go-control-plane v0.11.1-0.20230524094728-9239064ad72f
```

Comment:
should we add go-control-plane bump to the changelog too?

Reply:
Added